signingscript/src/signingscript/sign.py

```diff
@@ -537,22 +537,24 @@ def _get_mac_sigpath(from_):
 
 # _get_widevine_signing_files {{{1
 def _get_widevine_signing_files(file_list):
-    """Return a dict of path:signing_format for each path to be signed."""
+    """Return a dict of path:is_blessed for each path to be signed."""
     files = {}
     for filename in file_list:
-        fmt = None
         base_filename = os.path.basename(filename)
+        if base_filename not in _WIDEVINE_BLESSED_FILENAMES and base_filename not in _WIDEVINE_NONBLESSED_FILENAMES:
+            log.debug(f"{filename} is not a widevine signing file")
```

Comment:
That is going to be quite noisy...

Reply:
Yeah, that's fair. It's probably not necessary to add this in the end; we already log the other path.